components/admin_console/workspace-optimization/dashboard.tsx

```diff
@@ -64,6 +70,15 @@ const AccordionItem = styled.div<{iconColor: string}>`
     }
 `;
 
+const successIcon = (
+    <div className='success'>
+        <SuccessIconSvg
```

Comment:
is it possible to pass the `className` directly to the svg? that way we could avoid the extra div

Reply:
yes, it is possible, but the div is the one that handles the styles for the background and the radius that makes it look as a circle, and for handling the state changes where the accordion item list lenght is 0 and we switch between normal icon circle or the light green one for the success icon